prac2-pages/Siddharth_gautam_login_page/lib/main.dart

```diff
@@ -0,0 +1,198 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+
+      home: Scaffold(
+
+        body: const MyStatefulWidget(),
+
+      ),
+    );
+  }
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  const MyStatefulWidget({Key? key}) : super(key: key);
+
+  @override
+  State<MyStatefulWidget> createState() => _MyStatefulWidgetState();
+}
+
+class _MyStatefulWidgetState extends State<MyStatefulWidget> {
+  TextEditingController nameController = TextEditingController();
+  TextEditingController passwordController = TextEditingController();
+  bool value=false;
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+        padding: const EdgeInsets.all(20),
+        child: ListView(
+          children: <Widget>[
+
+            Container(
```

Comment:
Try to minimize the use of Containers in Flutter, as it is quite heavy on the resources. 

- It is also advised to go through the docs for the properties that you are using frequently, just to read them once and get a brief overview of the tasks supposedly performed by these properties.
- In this case, I would suggest you to see the **alignment** property in the docs. And judge for yourself, whether it is required or not. :thought_balloon: